tests/playbooks/tests_team.yml

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: team0
+    profile: "{{ interface }}"
+    lsr_fail_debug:
+      - __network_connections_result
+  tasks:
+    - debug:
+        msg: "this is: playbooks/tests_team.yml"
+      tags:
+        - always
+
+    - block:
+        - include_tasks: tasks/run_test.yml
+          vars:
+            lsr_description: I can create a profile
```

Comment:
The description should be  `Create a team interface with two ports`.

Reply:
my 2 cents: I try to use user stories, then the description would be "I can create a team interface with two ports".

Reply:
Oh that description is fine too.

Reply:
@ffmancera it seems that you propose not to attach any members in the other comment, then the description should be adjusted, too.